README.md

```diff
@@ -1593,4 +1595,44 @@ ensure that a Rich Text Formatted value is shown properly on the page. To use th
 ANSWERS.formatRichText(rtfFieldValue)
 ```
 
-For instance, this function can be used in the `dataMappings` of a Card to display an RTF attribute. When using this function, you must ensure that the relevant Handlebars template correctly unescapes the value's resultant HTML.
\ No newline at end of file
+For instance, this function can be used in the `dataMappings` of a Card to display an RTF attribute. When using this function, you must ensure that the relevant Handlebars template correctly unescapes the value's resultant HTML.
+
+# CSS Variable Styling
+
+The Answers SDK supports styling specific elements with CSS variables for runtime styling.
+* All sdk variables are exposed in the `:root` ruleset
+* All overridden variables must be included the `:root` ruleset
+* All overrides should be defined after Answers CSS is imported and before ANSWERS.init is called
+* To see available variables, see the scss modules in the SDK
+* For example, to override:
+
+```html
+<style>
+  :root {
+    --yxt-font-font-family: sans-serif;
+    --yxt-color-brand-primary: green;
+    --yxt-color-text-primary: #212121;
+    --yxt-searchbar-button-background-color-hover: red;
+    --yxt-nav-border-color: #e9e9e9;
+  }
+</style>
+```
+
+Most browsers have native css variable support. In legacy browsers, 
+we use the [css-vars-ponyfill](https://github.com/jhildenbiddle/css-vars-ponyfill).
+* If you opt-out of automatic resolution of variables, you should call ANSWERS.ponyfillCssVariables()
```

Comment:
Before this point, I would mention that we will automatically do the reconciliation for them. All they need to do is make sure things get loaded in the correct order.

Reply:
I think we should reiterate here that they can set that boolean to disable this.